common/persistence/dataInterfaces.go

```diff
@@ -232,9 +232,6 @@ type (
 		CancelRequestId                        string
 		StickyTaskQueue                        string
 		StickyScheduleToStartTimeout           *time.Duration
-		ClientLibraryVersion                   string
```

Comment:
Why are we removing it? Looks like useful info. Obviously, field names should change.

Reply:
After discussion with @samarabbas we decided to store client version info later in workflow history as version marker. Storing only last client version here doesn't make much sense.